tests/unit-tests/crud/data.php

```diff
@@ -273,4 +273,48 @@ function test_add_meta_data_overwrite_before_save() {
 		$object->update_meta_data( 'test_field_0', 'another field 2' );
 		$this->assertEquals( 'val1', $object->get_meta( 'test_field_2' ) );
 	}
+
+	/**
+	 * Test protected method set_date_prop by testing a order date setter.
+	 */
+	function test_set_date_prop() {
+		$object = new WC_Order();
+
+		// Change timezone in WP.
+		update_option( 'gmt_offset', -4 );
```

Comment:
@thenbrent I'll try it, but as you point out WP expects UTC server time, and the status report also errors for non-UTC time. So if this wasn't fixed for this release,  I don't think it would be critical. Additionally, date/strtotime is used heavily throughout WC and WP so wouldn't there be abnormalities already if it were set by a plugin somewhere?